indico/modules/rb_new/client/js/common/timeline/MonthlyTimelineContent.jsx

```diff
@@ -75,10 +75,28 @@ export default class MonthlyTimelineContent extends WeeklyTimelineContent {
         const nDays = this.dates.length;
         const daySize = (100 / nDays);
 
+        const {dateRange} = this.props;
+        const emptyDays = this.dates.filter((day) => {
+            return dateRange.length !== 0 && !dateRange.includes(day);
+        }).map((day) => {
+            return this.dates.findIndex((el) => el === day) + 1;
+        });
+
+        const additionalStyles = (dayNumber) => {
+            if (!emptyDays.includes(dayNumber)) {
+                return {};
+            }
+
+            return {
+                backgroundColor: '#FBE1E1',
```

Comment:
Could it be possible to keep this somewhere else in a variable? If someday we decide to change the palette it will be hard to change all these values. I wonder if it'd be possible to export sass variables to js... Ok, I found [this](https://www.bluematador.com/blog/how-to-share-variables-between-js-and-sass), it might be interesting to apply to the project, what do you think? cc: @pferreir @ThiefMaster @meluru @panagiotappl